deploy_snapshot_and_javadoc.sh

```diff
@@ -24,4 +25,26 @@ else
   echo "Deploying snapshot..."
   ./gradlew clean uploadArchives
   echo "Snapshot deployed!"
+
+  echo "Uploading javadoc..."
+
+  ./gradlew aggregatedJavadocs
+
+  {
+    set +x
+    cd $HOME
+    git config --global user.email "travis@travis-ci.org"
+    git config --global user.name "travis-ci"
+    git clone --quiet https://${GH_TOKEN}@github.com/wala/javadoc > /dev/null
```

Comment:
Enclose either the entire URL or just `$GH_TOKEN` in double quotes in case of oddball characters in `$GH_TOKEN`. Special characters seem unlikely here, but we should be extra careful with anything coming into a script from its environment.